gcp-gke-core-services/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "md_metadata" {
```

Comment:
I tried to make this interface (these variables) as easy as possible, thoughts?

Reply:
I love the ease of this, however, I think we should make modules as massdriver agnostic as possible. A user not on massdriver would have to dig through source to figure out what this needs to be. Lets make our root bundles munge the data and the modules completely massdriver agnostic.

Reply:
We chatted "offline" about this. Totally onboard with flattening the module interfaces and making them less or not at all Massdriver specific. For this one, `md_metadata` is sprinkled throughout, consumed by other modules inside of here. So, we'll come back to this one later.